sympy/physics/mechanics/kane.py

```diff
@@ -117,7 +118,7 @@ class KanesMethod:
     def __init__(self, frame, q_ind, u_ind, kd_eqs=None, q_dependent=None,
             configuration_constraints=None, u_dependent=None,
             velocity_constraints=None, acceleration_constraints=None,
-            u_auxiliary=None):
+            u_auxiliary=None, bodies=None, forcelist=None):
```

Comment:
Docstring about these new kwargs is needed.

Reply:
@moorepants  I don't think all other `__init__`  kwargs are documented.

Reply:
That's unfortunate!